src/components/ClaimReview/TopicInput.tsx

```diff
@@ -81,7 +81,7 @@ const TopicInput = ({ sentence_hash, topics }) => {
                             color: colors.bluePrimary,
                         }}
                     >
-                        <EditFilled />
+                        {tags.length ? <EditFilled /> : <PlusOutlined />}
```

Comment:
Show nothing if the user is not logged in

Reply:
@thesocialdev  this was already done in the last PR

Reply:
![image](https://user-images.githubusercontent.com/73478823/186700732-b951395d-a78a-4546-83bd-1e655f14197d.png)